tests/Samples.Server.Tests/Serializer.cs

```diff
@@ -29,17 +30,17 @@ internal static FormUrlEncodedContent ToFormUrlEncodedContent(GraphQLRequest req
 
             if (request.OperationName != null)
             {
-                dictionary[GraphQLRequest.OPERATION_NAME_KEY] = request.OperationName;
+                dictionary["operationName"] = request.OperationName;
```

Comment:
Here in the tests also we see another reference to the JSON keys, if you will.  As these are tests, I just embedded them directly in the source, which is probably as it should be anyway.